src/components/Register.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+function Register() {
+  const [newEmail, setNewEmail] = useState('');
+  const [newPassword, setNewPassword] = useState('');
+  const [error, setError] = useState(null);
+
+  async function registerForm(e) {
+    e.preventDefault();
+    try {
+      const response = await fetch(`https://localhost:8000/api/v1/auth/register`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          username: newEmail,
+          password: newPassword,
+          email: newEmail
+        })
+      });
+
```

Comment:
I think we need to parse the response before we do any error handling. I think ```if (!response.ok)``` will throw an error or always eval to false otherwise. The pattern I'm familiar with goes like this:
```response = await fetch(...)
data = response.json()

if (data.error) or maybe (data.status !=200) {
... do some error handling
}

 whatever logic you need when a new user is successfully created goes here
```